### PR TITLE
fix vicuna example

### PR DESCRIPTION
**Pull request by infwinston**
Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: llm/vicuna/scripts/sync_local_checkpoint.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        num_local_ckpt=$(ls ${local_path} | grep checkpoint- | wc -l)
+        echo "num_local_ckpt: ${num_local_ckpt}"
+        if [ ${num_local_ckpt} -gt $MAX_NUM_CKPT ]; then
+            for ckpt in $(ls ${local_path} | grep checkpoint- | grep -E '[0-9]+' | sort -t'-' -k1,1 -k2,2n | head -n-${MAX_NUM_CKPT}); do
+                rm -rf ${local_path}/${ckpt}
+            done
+        fi
+    fi
+
+    sleep 600
```
Comment (Michaelvll) [inline llm/vicuna/scripts/sync_local_checkpoint.sh:27]: Should we upload the whole local directories anyway or do you think we still need this uploading last checkpoint logic?

**Review by Michaelvll — COMMENTED** [review]
Thanks for adding the fix @infwinston! Left a question.

**Review by Michaelvll — APPROVED** [review]
It should be fine to let this PR in first, and fix the comment above later. : )

**Comment by infwinston** [comment]
Sorry I didn't get time to further improve this. Let's fix it first and hopefully we have a more robust sync mode by @landscapepainter's https://github.com/skypilot-org/skypilot/pull/2190.

**Pull request merged**